### PR TITLE
Adding repo overrides to supply missing images and bump test coverage

### PR DESCRIPTION
**Pull request by jaypoulz**

### CODE REVIEW

File: ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml [hunk 1]
```diff
@@ -513,18 +517,22 @@ objects:
...
         mock-nss.sh openshift-install create manifests --dir=/tmp/artifacts/installer
         sed -i '/^  channel:/d' /tmp/artifacts/installer/manifests/cvo-overrides.yaml
+        
+        # Bump the libvirt masters memory to 16GB
+        export TF_VAR_libvirt_master_memory=16384
         ls /tmp/artifacts/installer/openshift
         for ((i=0; i<$MASTER_REPLICAS; i++))
         do
-            sed -i -e 's/7168/16384/g' /tmp/artifacts/installer/openshift/99_openshift-cluster-api_master-machines-${i}.yaml
+            yq write --inplace /tmp/artifacts/installer/openshift/99_openshift-cluster-api_master-machines-${i}.yaml spec.providerSpec.value.domainMemory 16384
```
Comment (Prashanth684) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml:528]: needs to be `spec.providerSpec.value[domainMemory]`

File: ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml [hunk 2]
```diff
@@ -513,18 +517,22 @@ objects:
...
         done
         
         # Bump the libvirt workers memory to 8GB
-        sed -i -e 's/7168/8192/g' /tmp/artifacts/installer/openshift/99_openshift-cluster-api_worker-machineset-0.yaml
+        yq write --inplace /tmp/artifacts/installer/openshift/99_openshift-cluster-api_worker-machineset-0.yaml spec.template.spec.providerSpec.value.domainMemory 8192
```
Comment (Prashanth684) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-remote-libvirt-e2e.yaml:532]: same comment as above

**Comment by jaypoulz** [comment]
/assign @crawford 

The latest run proves that this is at least as stable as the tests currently running. We are looking into how to fix the intermittent network test failures, since they've been inconsistent since we first landed our 4.2 job.

**Pull request merged**